brainbox/modeling/utils.py

```diff
@@ -98,20 +100,22 @@ def fit(self, progress=False):
             masks_set = maskdf.groupby(self.features.tolist()).groups
             for current_mask in tqdm(masks_set, desc='feature subset', leave=False):
                 cells = masks_set[current_mask]
-                new_feature_idx, nf_score = self._get_best_new_feature(current_mask, cells)
+                new_feature_idx, nf_score, all_scores = self._get_best_new_feature(current_mask, cells)
```

Comment:
I mentioned this in a message on the PR, but here you will be redefining the `all_scores` every loop, so you will only store a df with the final set of candidates for each unit. The rest of the values will be NaN. Is that what you wanted? Or would you prefer something like a stack of DFs for each step containing all of the intermediate scores?